scaleway/sentry.go

```diff
@@ -0,0 +1,58 @@
+package scaleway
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/getsentry/raven-go"
+)
+
+const (
+	dsn = "https://02b954a634fc439dbb1ac16df9cd3da5@sentry.internal.scaleway.com/223"
+)
+
+// RecoverPanicAndSendReport will recover error if any, log them, and send them to sentry.
+// It must be called with the defer built-in.
+func RecoverPanicAndSendReport() {
+	e := recover()
+	if e == nil {
+		return
+	}
+
+	sentryClient, err := newSentryClient()
+	if err != nil {
+		log.Printf("cannot create sentry client: %s", err)
+	}
+
+	err, isError := e.(error)
+	if isError {
+		logAndSentry(sentryClient, err)
```

Comment:
I think error logging is handled by the terraform binary directly as we did not handle it before.